src/m_obj.c

```diff
@@ -919,6 +920,31 @@ int obj_issignaloutlet(const t_object *x, int m)
     "MAINSIGNALIN" has been provided, in which case the object won't
     promote scalars correctly.  Nonetheless we provide it so that at least
     such a badly written object won't crash Pd. */
+
+void obj_sigoutletsetchans(const t_object *x, int m, int nchans)
+{
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+    if (o2 && (o2->o_sym == &s_signal))
+    {
+        o2->o_nchans = nchans;
+    }
+}
+
+int obj_sigoutletgetchans(const t_object *x, int m)
+{
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+
+    if (o2 && (o2->o_sym == &s_signal)){
```

Comment:
Again, open braces go on their own lines. But in this case, no braces :) (Also in `obj_sigoutletssetchans`.)

Sorry, for the nitpicking, but we really try to keep the code style consistent. Soon it will be in your muscle memories :)

Reply:
No problem, I will fix it